### DISCUSSION TITLE
CreateInvoice

### DISCUSSION THREAD

**Discussion by bettialessandro** [fattureincloud/api#141] [Q&A] [answered]
Buongiorno , 
esiste una guida più dettagliata per i singoli campi in fattura ? Nella fattispecie riesco a  creare fatture e clienti facilmente ma non trovo dettagli per quanto riguarda le fatture extra cee con applicazione degli articoli in detrazione dell'IVA . Quali sono i campi preposti e come si articola una fattura senza iva ?  Nella vecchia versione avevo mappato il boolean prezzi_ivati = true o false specificando poi nelle note l'Articolo di esenzione oppure la lettera di intenti .. nella versione  2e  nell'SDK C# come  si procede in merito ? 
Grazie in anticipo  A chiunque mi dia delle info :-)

**Comment by emajo** [comment, marked as answer]
Ciao @bettialessandro ti consiglio di leggere [la guida sulla creazione della fattura](https://developers.fattureincloud.it/docs/guides/invoice-creation) dove troverai tutte le risposte che cerchi, in caso avessi ancora qualche dubbio non esitare a scriverci.

Grazie,
Emanuele

**Comment by bettialessandro** [comment]
Grazie ho trovato tutto  :-)
